magnum_capi_helm/tests/test_driver.py

```diff
@@ -1094,6 +1094,52 @@ def test_get_chart_version_from_template(self):
 
         self.assertEqual("1.42.0", version)
 
+    def get_cluster_helm_standard_values(self):
```

Comment:
Nit: I would prefer _get_cluster_helm_standard_values(), but otherwise this seems good to me.

Reply:
Good point, thanks; done as part of https://github.com/stackhpc/magnum-capi-helm/pull/23